components/NetworkStatusItem.qml

```diff
@@ -68,17 +68,46 @@ Rectangle {
             }
 
             Image {
+                visible: !appWindow.isMining
                 anchors.top: parent.top
-                anchors.topMargin: 6
+                anchors.topMargin: 6 * scaleRatio
                 anchors.right: parent.right
-                anchors.rightMargin: 11
+                anchors.rightMargin: 11 * scaleRatio
                 source: {
                     if(item.connected == Wallet.ConnectionStatus_Connected){
```

Comment:
Better to have single Image instance and add update `source` and `onClicked` to handle the mining status changes

Reply:
Noted. Thanks.